runtime/src/accounts_index.rs

```diff
@@ -32,7 +32,167 @@ pub const ITER_BATCH_SIZE: usize = 1000;
 
 pub type SlotList<T> = Vec<(Slot, T)>;
 pub type SlotSlice<'s, T> = &'s [(Slot, T)];
-pub type Ancestors = HashMap<Slot, usize>;
+
+pub type AncestorsForSerialization = HashMap<Slot, usize>;
+
+#[derive(Debug, Default, Clone, PartialEq, Serialize, Deserialize, AbiExample)]
+pub struct Ancestors {
```

Comment:
How about Ancestors in it's own module

Reply:
Yep, I was thinking about its own module earlier today. Seems appropriate.
I just went over all this with @carllin, too.